span-normalizer/span-normalizer/src/main/java/org/hypertrace/core/spannormalizer/jaeger/JaegerSpanToAvroRawSpanTransformer.java

```diff
@@ -24,9 +24,13 @@
   private static final Logger LOGGER = LoggerFactory
       .getLogger(JaegerSpanToAvroRawSpanTransformer.class);
 
-  private static final String VALID_SPAN_RECEIVED_COUNT = "valid.span.received.count";
+  private static final String SPANS_COUNTER = "hypertrace.reported.spans";
+  private final ConcurrentMap<String, Counter> statusToSpansCounter = new ConcurrentHashMap<>();
```

Comment:
Do we need concurrent map? The processor code is executed per stream thread.

Reply:
Make sense, will convert it to a static map as they are referring to the same metric.

Reply:
done